pkg/eks/services/eks.go

```diff
@@ -0,0 +1,89 @@
+package services
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/eks"
+)
+
+type EKSServiceInterface interface {
```

Comment:
Any reason we can't use the interfaces already defined for these services? For example, this is the EKS interface: https://github.com/aws/aws-sdk-go/blob/main/service/eks/eksiface/interface.go

One reason I can think of is that if we upgrade to the AWS Go SDK v2 it doesn't have these interface files.

Reply:
I'd like us to have more control here and be sure that the interfaces we use are always compatible with AWS SDK.